### PR TITLE
Contribute band design

### PR DESCRIPTION
**Pull request by oraNod**
Fixes issue #123

### CODE REVIEW

**Comment by oraNod** [comment]
@tiyiprh Hi, I can't seem to add you as a reviewer but could you please take a look at the changes here?

**Comment by tiyiprh** [comment]
@oraNod I would just follow the link that's pasted in the other issue correct?

**Comment by oraNod** [comment]
> @oraNod I would just follow the link that's pasted in the other issue correct?

Hi @tiyiprh Yes, that's fine. You can find the link for each pull request under the checks section in the "Read the Docs" build.

image

File: templates/homepage-contribute-band.tmpl [hunk 1]
```diff
@@ -6,24 +6,20 @@
...
         {% for item in homepage.contribute.ways %}
         <ul>
-          <li><i class="fa fa-star" aria-hidden="true"></i>&ensp;{{ item }}</li>
+          <li>{{ item }}</li>
         </ul>
         {% endfor %}
+        <p>{{ homepage.contribute.main }}</p>
+      </div>
+      <div class="contribute-box-right">
+        <img src="/images/bull-builder.svg" alt="Ansible bull builder." />
```
Comment (GregSutcliffe) [inline templates/homepage-contribute-band.tmpl:20]: Seems slightly too low for me?

image

Shouldn't the bull's feet be aligned with the last line of text?

Comment (GregSutcliffe) [reply templates/homepage-contribute-band.tmpl:20]: Also it feels odd to not have a link top-right, like the "View all Ansible events" below it or "View more" blogs above it. For consistency should all the bands have one?

File: data/homepage.yaml [hunk 2]
```diff
@@ -131,10 +131,11 @@ contribute:
     - Code a new module or fix a bug.
     - Improve the documentation.
   main: >-
-    If you want to get involved, come talk to us.
+    Want to get involved?
+    Come talk to us.
     We're friendly.
     And we have contributor resources to help you.
-  link: https://forum.ansible.com/pub/how-to-contribute
+    Click <a href="https://forum.ansible.com/pub/how-to-contribute" target="_blank">here</a> to learn more!
```
Comment (GregSutcliffe) [inline data/homepage.yaml:138]: HTML in the YAMl feels weird....

**Review by GregSutcliffe — DISMISSED** [review]
I'll wait for @tiyiprh to review too, but mostly this seems good to me. Couple of nits, though I'll merge without worrying about them if Tina ack's it.

File: templates/homepage-contribute-band.tmpl (hunk 1 above)
Comment (oraNod) [reply templates/homepage-contribute-band.tmpl]: Thanks I've fixed both of these and it's much better. Needs a media query for responsiveness but I'll do that in a follow up.

File: data/homepage.yaml (hunk 2 above)
Comment (oraNod) [reply data/homepage.yaml]: Yeah it was late at night. Way better to put this in a "learn more" section and keep section links in a consistent place.

**Comment by oraNod** [comment]
> I'll wait for @tiyiprh to review too, but mostly this seems good to me. Couple of nits, though I'll merge without worrying about them if Tina ack's it.

@GregSutcliffe I'll respect the decision to wait as I did add Tina as a reviewer. However she did ack in the issue here: https://github.com/ansible-community/community-website/issues/123#issuecomment-1716170799

I'd like to go ahead and merge if you're happy with the recent changes.

**Pull request merged**